concourse.yml

```diff
@@ -34,6 +34,7 @@ meta:
     disk:
       size: ~
       type: ~
+      encrypted: ~
```

Comment:
@jmcarp It's not necessary to provide an option to merge encryption here, since later, in line 266, you specifically set it to true, instead of grabbing this meta value. I would remove this, and the bit from the secrets example.